app/Models/Seat.php

```diff
@@ -13,7 +13,7 @@ class Seat extends Model
 {
     use HasFactory;
 
-    protected $table = 'r2023_seats';
+    protected $table = 'r2024_seats';
```

Comment:
maybe use it only like 'seats' and 'reservations', for GDPR purposes and maintainability